src/main/steps/check-jurisdiction/content.ts

```diff
@@ -0,0 +1,30 @@
+import { TranslationFn } from '../../app/controller/GetController';
+import { FormContent } from '../../app/form/Form';
+
+export const generateContent: TranslationFn = ({ isDivorce }) => {
+  const en = {
+    title: `Check if you can ${isDivorce ? 'get a divorce' : 'end your civil partnership'} in England and Wales`,
+    line1: `You must have some connection to England or Wales for the courts to have the legal power to ${
+      isDivorce ? 'grant you a divorce' : 'end your civil partnership'
+    }. This legal power is known as ‘jurisdiction’.`,
+    line2: 'The following questions will find out what connections you have to England or Wales.',
+  };
+
+  // @TODO translations
+  const cy: typeof en = {
+    ...en,
+  };
+
+  const common = {
+    form,
```

Comment:
Where is this defined? Does this page have any fields? If so they need to be mapped to the API

Reply:
`form` is defined just below, there are no fields for this page, but there is a "Continue" button. Having an empty `fields` object renders the page correctly without any fields and with the "Continue" button at the bottom.